.github/workflows/commit_epm.yml

```diff
@@ -0,0 +1,74 @@
+name: Run epm with skeleton
+
+# Controls when the action will run. Triggers the workflow on pull request labeled testing-before-checkin.
+on:
+  pull_request_target:
+    types: labeled
+
+jobs:
+  rune_epm:
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'testing-before-checkin') }}
+    runs-on: ${{ matrix.sgx }}
+    strategy:
+      matrix:
+        sgx: [self-hosted, SGX2]
+        tag: [ubuntu18.04, centos8.1]
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: refs/pull/${{ github.event.pull_request.number }}/merge
+
+    - name: Prepare work
+      run: echo "CPU_NUM=$(nproc --all)" >> $GITHUB_ENV
+
+    - name: Create container
+      run: |
+        rune_test=$(docker run -itd --privileged --rm --net host --device /dev/sgx/enclave --device /dev/sgx/provision -v $GITHUB_WORKSPACE:/root/inclavare-containers rune-test:${{ matrix.tag }});
+        echo "rune_test=$rune_test" >> $GITHUB_ENV
+
+    - name: Build install packages and run epm service
+      run: |
+        docker exec $rune_test bash -c "mkdir -p /var/run/epm;
+        mkdir -p /var/epm;
+        mkdir -p /var/local/epm;
+        cp -r /root/inclavare-containers /root/inclavare-containers-test;
+        cp /root/inclavare-containers-test/epm/conf/config.toml /var/epm/;
+        cd /root/inclavare-containers-test;
+        make -j${CPU_NUM} && make install -j${CPU_NUM}";
```

Comment:
You also need to generate the configuration file `/var/run/epm/config.toml` of `epm`. So it is better to use `make package` to generate rpm and deb packages to install rather than `make binary`.